package.json

```diff
@@ -28,6 +28,7 @@
     "streamx": "^2.18.0"
   },
   "devDependencies": {
+    "bare-http1": "^3.8.0",
```

Comment:
This creates a cycle between `bare-stream` and `bare-http1` so let's not test against that. You want to test streams in isolation anyway, so mock out `Readable`/`Writable` streams as appropriate.